tests/unit/opensearch_utils/test_client.py

```diff
@@ -122,6 +89,13 @@ def test_send_document_create(self):
         self.client.send_document(document)
         document_out = self.client.get_document(document)
 
+        # openmock does not output the _primary_term and _seq_no. These are two
+        # opensearch fields that we're not using. Adding these fields to the output to
+        #  highlight that these do exist in a real opensearch document, but are missing
+        # from openmock. These missing fields do not impact the test
```

Comment:
Thanks for comments like this and PR overview. Really helpful to understand what's being changed.